### PR TITLE
Added Instructions for Apache Reverse Proxy

### PR DESCRIPTION
**Pull request by brian-olson**

### CODE REVIEW

**Comment by brian-olson** [comment]
Added information on WinRM issues I experienced using Packer v0.8.2. Compiling the latest code (0.8.3) from the Github repo resolves all issues.

File: admin.adoc [hunk 1]
```diff
@@ -404,6 +404,70 @@ TBD. User contributions welcome.
 Using the wsgi hasn't been thoroughly tested. If you test, please send feedback
 to the dev list and we can try and fix things.
 
+Running the GRR HTTP Server Behind Apache Reverse Proxy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```
Comment (destijl) [inline admin.adoc]: Add a short explanation here about why you would want to do this. Something like:

```
Running apache as a reverse proxy in front of the GRR admin UI is a good way to provide SSL protection for the UI traffic and also integrate with corporate single-sign (if available) on for authentication.
```

File: admin.adoc [hunk 2]
```diff
@@ -404,6 +404,70 @@ TBD. User contributions welcome.
...
+Generate or otherwise obtain SSL certificates
```
Comment (destijl) [inline admin.adoc]:
```
Buy an SSL certificate, or link:https://www.digitalocean.com/community/tutorials/how-to-create-a-ssl-certificate-on-apache-for-ubuntu-14-04[generate] one if you're only testing.
```

Then delete the following sentence.

File: admin.adoc [hunk 3]
```diff
@@ -404,6 +404,70 @@ TBD. User contributions welcome.
...
+</VirtualHost>
+-----------------------------------------
+
+Restart apache
+
+-----------------------------------------
+service apache2 restart
+-----------------------------------------
+
+* NOTE: This reverse proxy will only proxy the AdminUI. It will have no impact on the agent communications on port 8080. It is advised to restrict access to the AdminUI.
```
Comment (destijl) [inline admin.adoc]: "access to the AdminUI at the network level"

**Comment by destijl** [comment]
This is great, thanks for the contribution, some minor suggestions.

File: windowsclient.adoc [hunk 4]
```diff
@@ -7,7 +7,7 @@ automated as possible.
 == Client Build VM ==
 
 First, install link:https://www.vagrantup.com/[Vagrant],
-link:https://packer.io/[Packer], and VirtualBox.  See
+link:https://github.com/mitchellh/packer/[Packer] (ensure you download and compile version 0.8.3 at min to avoid WinRM issues), and VirtualBox.  See
```
Comment (destijl) [inline windowsclient.adoc]: I think the original link is a better place to point people to. They may move that github repo out of that personal directory at some stage. Download is one click away from the packer.io page.

**Comment by destijl** [comment]
Hey brian, I'm just going to merge and make these edits myself. Thanks for writing it up.

**Pull request merged**